msg/ObjectHypothesisWithPose.msg

```diff
@@ -1,18 +1,12 @@
-# An object hypothesis that contains position information.
+# An object hypothesis that contains pose information.
 
-# The unique ID of the object class. To get additional information about
-#   this ID, such as its human-readable class name, listeners should perform a
-#   lookup in a metadata database. See vision_msgs/VisionInfo.msg for more detail.
-string id
-
-# The probability or confidence value of the detected object. By convention,
-#   this value should lie in the range [0-1].
-float64 score
+# The object hypothesis (ID and score).
+ObjectHypothesis hypothesis
```

Comment:
I'd like a different name for this, but honestly I can't think of a better option